projects/igniteui-angular/src/lib/chips/chips-area.spec.ts

```diff
@@ -139,7 +139,7 @@ describe('IgxChipsArea', () => {
         const chipArea = fix.debugElement.queryAll(By.directive(IgxChipsAreaComponent));
         const chipAreaComponent = fix.componentInstance;
 
-        expect(chipArea[0].nativeElement.className).toEqual('igx-chip-area');
+        expect(chipArea[0].nativeElement.className).toEqual('igx-chip-area customClass');
```

Comment:
I see the fix is applicable to the `IgxChipAreaComponent`. Is the fix for the `IgxChipComponent` already applied?